azure-pipelines.yml

```
@@ -1,8 +1,16 @@
 trigger:
 - main
+  paths:
+    exclude:
+    - subscriptions.json
```

Are you sure this works❔ Might need to be
```suggestion
    - /Maestro/subscriptions.json
```

---

Yup, looks like I was off by a directory